### DISCUSSION TITLE
Shouldn't @parcel/transformer-less use parcels resolver for resolving paths?

### DISCUSSION THREAD

**Discussion by code-and-such** [parcel-bundler/parcel#7606] [Q&A]
I'm trying to extract some code from an application to an NPM packet. Part of what I extract are less files, which are used by react components

In the less files there are two kind of references: 

`@import (reference) 'something_in_node_modules/styles.less';`

and and this kind 

```
  &:hover > span.t2-icon {
    background-image: url('something_in_node_modules/vectorgraphics.svg');
  }
```

It seems like @parcel/transformer-less is [resolving the imports,](https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/less/src/LessTransformer.js) while the normal parcel resolver mechanism resolves the other kind. 
 
Let's say I want to use a resolver to implement webpack's tilde syntax (~my_node_module/....). That can be implemented with parcels resolver mechanism, but since that one isn't used for imports, but instead the transformer's internal one, I get this in my face:
```
@parcel/transformer-less: The @import path "~my_node_module/styles" is using webpack specific syntax, which isn't supported by Parcel.
``` 

To work around this and get both kind of external references (shown above) to look the same I need to use both a custom resolver (@mischnic/parcel-resolver-root) and a .lessrc. I can explain more if needed, but I think the fact that there is resolving written directly in the plugin kinda shows what I'm hitting at 

https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/less/src/LessTransformer.js

Before I open an issue, I would like to know whether this is the desired behaviour? In the sass resolver I can it it falls back on parcels mechanism 
https://github.com/parcel-bundler/parcel/blob/74fcc3fbe80cea993bff10e158df3d0f701973c7/packages/transformers/sass/src/SassTransformer.js#L160

Also ... The Sass one seems to be handling tilde which the less one explicit refuse to do.

**Comment by devongovett** [comment]
Here, it does use Parcel's resolver, after it tried the default Less one: https://github.com/parcel-bundler/parcel/blob/c3a28caf3dd7cd44a092b3d1361e8d1ac26e04ee/packages/transformers/less/src/LessTransformer.js#L155

Less treats bare imports as relative paths by default. If you'd like to refer to a node module, you can use the `npm:` scheme: https://parceljs.org/blog/v2/#new-npm%3A-scheme
